CONTRIBUTING.md

```diff
@@ -317,6 +317,7 @@ New features require a +1 from 2 other core committers (besides yourself).
 ### Process: Roadmap
 
 The roadmap located [here](https://github.com/cookiecutter/cookiecutter/milestones?direction=desc&sort=due_date&state=open)
+TODO: change road map to cookieninja
```

Comment:
Let's just update this to our milestones page

Reply:
Fixed